src/Admin/SiteHealth.php

```diff
@@ -76,13 +66,24 @@ public function __construct( MonitorCssTransientCaching $css_transient_caching,
 	 * Adds the filters.
 	 */
 	public function register() {
-		add_filter( 'site_status_tests', [ $this, 'add_tests' ] );
-		add_filter( 'debug_information', [ $this, 'add_debug_information' ] );
-		add_filter( 'site_status_test_result', [ $this, 'modify_test_result' ] );
-		add_filter( 'site_status_test_php_modules', [ $this, 'add_extensions' ] );
 
-		add_action( 'admin_print_styles-tools_page_health-check', [ $this, 'add_styles' ] );
-		add_action( 'admin_print_styles-site-health.php', [ $this, 'add_styles' ] );
+		if ( is_admin() ) {
```

Comment:
The `is_admin()` test I think will cause problems in case Site Health runs during wp-cron. There is weekly `wp_site_health_scheduled_check` scheduled event which runs [`WP_Site_Health::wp_cron_scheduled_check()`](https://github.com/WordPress/wordpress-develop/blob/d82accc291317ad590c1e81be9d914e5cec42d90/src/wp-admin/includes/class-wp-site-health.php#L2723-L2832). Since `is_admin()` is used here the `site_status_tests` filter wouldn't be added so the tests would not be included when running in WP Cron. I'll remove this.